terraform/bastion/Jenkinsfile-check

```diff
@@ -0,0 +1,54 @@
+library("tdr-jenkinslib")
+def repo = "tdr-scripts"
+
+pipeline {
+  agent {
+    label "master"
+  }
+  parameters {
+    choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are checking the age of")
+  }
+  stages {
+    stage("Run git secrets") {
+      steps {
+        script {
+          tdr.runGitSecrets(repo)
+        }
+      }
+    }
+    stage('Destroy bastion if the instance is older than 7 days') {
+      agent {
+        ecs {
+          inheritFrom 'aws'
+          taskrole "arn:aws:iam::${env.MANAGEMENT_ACCOUNT}:role/TDRJenkinsRunEC2DescribeInstances${params.STAGE.capitalize()}"
+        }
+      }
+      stages {
+        stage('Run delete user script') {
+          steps {
+            script {
+              def accountNumber = tdr.getAccountNumberFromStage(params.STAGE)
+              def tooOld = sh(script: "python3 /check_ec2_instance_age.py ${accountNumber} ${params.STAGE} bastion-ec2-instance-${params.STAGE} 7", returnStdout: true).trim()
```

Comment:
Where did the 7 days come from? I'm not sure what it should be, but I'd prefer it to be much shorter if possible. Ideally a few hours, but maybe a day or two if that would cause problems.

Reply:
I just picked a number :smile: I'll change it to a day for now and see how we get on.